client/mass/store.js

```diff
@@ -507,10 +507,8 @@ function validatePlot(p, vocabApi) {
 }
 
 function validateGenericPlot(p, vocabApi) {
-	// console.log(p)
-	if (!p.term) throw 'plot error: plot.term{} not an object'
 	try {
-		validatePlotTerm(p.chartType == 'regression' ? p.outcome : p.term, vocabApi)
+		if (p.term) validatePlotTerm(p.chartType == 'regression' ? p.outcome : p.term, vocabApi)
```

Comment:
need to get rid of validateGenericPlot() from store, as this is legacy and totally outdated, and headache for us adding new plots